src/lib_ccx/ccx_decoders_structs.h

```diff
@@ -62,7 +62,7 @@ typedef struct eia608_screen // A CC buffer
 {
 	/** format of data inside this structure */
 	enum ccx_eia608_format format;
-	unsigned char characters[15][33];
+	char characters[15][33];
```

Comment:
This is a _major_ change in a structure that has been stable for years. Why do you want to do this? There's a reason for unsigned.